msibi/tutorials/README

```diff
@@ -5,4 +5,9 @@ code works as expected.
 LJ : recover known LJ potential
     - This test, as described in the original paper, is recovering a
       known potential to show that the methodology works. The target
-      data was derived from LJ fluids at three different states. 
+      data was derived from LJ fluids at three different states.
+
+PROPANE : recover known propane potential
+    - This test, as described in the original 2014 paper, is recovering a
+      known potential to show that the methodology works. The target
```

Comment:
How would the README for this section be rephrased? How about `This test, as described in the original 2014 paper, performs further optimizations on propane using a coarse-graining 3-1 mapping. The 3-site propane is mapped to the single-site propane and the generated data can be compared to known LJ graphs from literature.`